ui/bone_root.py

```diff
@@ -6,6 +6,26 @@
 from ..tools.translations import t
 
 
+@register_wrap
+class SearchMenuOperator_root_bone(bpy.types.Operator):
+    bl_description = t('Scene.root_bone.desc')
+    bl_idname = "scene.search_menu_root_bone"
+    bl_label = ""#t('Scene.root_bone.label')
+    bl_property = "my_enum"
+
+    my_enum: bpy.props.EnumProperty(name=t('Scene.root_bone.label'),description= t('Scene.root_bone.desc'), items=Rootbone.get_parent_root_bones)
```

Comment:
Blender 2.79 uses Python 3.5, which does not support variable annotations (the file will not be parsed as valid Python), you need to use the old style of `my_enum = bpy.props.EnumProperty` and then the `@register_wrap` will result in the necessary annotations for newer Blender versions that expect properties to be defined via annotations.